google/api/BUILD.bazel

```diff
@@ -0,0 +1,246 @@
+# Copyright 2018 Google LLC
```

Comment:
Nit: 2019

Reply:
This is not a new file, it is a sync from master, where it is 2018.